docs/src/modules/components/ApiPage.js

```diff
@@ -207,7 +207,7 @@ function ApiDocs(props) {
   const description = t('api-docs.pageDescription').replace(/{{name}}/, componentName);
 
   const source = filename
-    .replace(/\/packages\/mui(-(.+?))?\/src/, (match, dash, pkg) => `@mui/${pkg || 'core'}`)
+    .replace(/\/packages\/mui(-(.+?))?\/src/, (match, dash, pkg) => `@mui/${pkg || 'material'}`)
```

Comment:
```suggestion
    .replace(/\/packages\/mui(-(.+?))?\/src/, (match, dash, pkg) => `@mui/${pkg}`)
```

Wouldn't this work? All packages now are after "mui-"

Reply:
ah yes, that's better.